spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
This template test is showing as a "Pending" test. We should either remove it, or create a passing test.

Reply:
Ah, thank you! Great catch! For now I am going to remove the line, but leave the file in case we use it later on.